src/main/java/dev/jaxcksn/nanoleafMusic/controllers/ConnectToDevice.java

```diff
@@ -90,13 +93,14 @@ public String toString() {
         }
     }
 
-    private List<AuroraMetadata> findNanoleaf() throws IOException {
-        List<AuroraMetadata> auroras = new ArrayList<>();
+    private List<NanoleafDeviceMeta> findNanoleaf() throws IOException {
+        List<NanoleafDeviceMeta> auroras = new ArrayList<>();
 
         try {
             Service service = Service.fromName("_nanoleafapi._tcp");
             Query query = Query.createFor(service, Domain.LOCAL);
             Set<Instance> instances = query.runOnce();
+            auroras.add(new NanoleafDeviceMeta("192.168.0.20", 16021 ,"xxx","Shapes.Hexagon"));
```

Comment:
Forgot to remove this before merging. Might need to fix this quickly

Reply:
Open an issue and then make a PR for it real quick to the Java branch